client/core/core.go

```diff
@@ -485,11 +487,13 @@ func (c *Core) Exchanges() map[string]*Exchange {
 	infos := make(map[string]*Exchange, len(c.conns))
 	for host, dc := range c.conns {
 		infos[host] = &Exchange{
-			Host:       host,
-			Markets:    dc.markets(),
-			Assets:     dc.assets,
-			FeePending: dc.acct.feePending(),
-			Connected:  dc.connected,
+			Host:          host,
+			Markets:       dc.markets(),
+			Assets:        dc.assets,
+			FeePending:    dc.acct.feePending(),
+			Connected:     dc.connected,
+			ConfsRequired: uint32(dc.cfg.RegFeeConfirms),
+			RegConfirms:   dc.regConfirms,
```

Comment:
`refreshUser`=>`Exchanges` reading `dc.regConfirms` can happen in many places, but `regConfirms` is not guarded (`connMtx` is for `Core.conns`, not for these fields of `dexConnection`).